app/templates/events/trackVolunteers.html

```diff
@@ -59,6 +59,7 @@ <h1 class="text-center">{{event.name}}</h1>
         <th>Email</th>
         <th>Phone Number</th>
         <th class="noSort">Completed Trainings?</th>
+        <th class="noSort">Eligible/Not Banned?</th>
```

Comment:
We do not like this title, the slash is unnecessary, I would recommend changing it to either eligible or eligibility and having what's in the column be like is on the volunteer profile where it says eligible or banned.

Reply:
Changed to Eligible